emails/views.py

```diff
@@ -358,6 +383,35 @@ def _sns_message(message_json):
     )
```

Comment:
Technically, while relaying the email and we get a bounce response does that mean we response to the original sender about the bounce reply with the receiver's real address instead of the relay address? If so, we should probably check for the bounce email and replace that information?

Reply:
I thought about this a bit more. In https://github.com/mozilla/fx-private-relay/issues/79 we plan to create a receipt rule to permanently hard bounce to the original sender for **_deleted aliases_**, which will inform original senders with appropriate bounces.

So the question is what we do when the **_real recipient's email address_** is bounce ...

If it's a soft bounce, we **_don't_** want to "relay" that bounce back to the original sender, as that would leak temporary state information about the real recipient's email address.

If it's a hard bounce (for >X days), we may want to set their aliases to disabled, and create a receipt rule to soft bounce emails sent to their aliases. 

But I think we don't ever want to **_permanently delete_** aliases (and hard bounce emails sent to them) based on some heuristics. We should only permanently delete aliases when the user explicitly permanently deletes them.

Reply:
@groovecoder can we create a separate issue to document this? I am happy to just copy paste what you wrote into a separate issue and reference this PR.